py-scripts/scripts_deprecated/tip-cicd-sanity/Throughput_Test.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa
```

Comment:
wants *required by* or shebang

Reply:
This script is deprecated. I can add this, but unlikely needed given that it's on its way out.

Reply:
deprecated, yeah, don't care